src/lib/mongoBackend/safeBsonGet.cpp

```diff
@@ -105,8 +102,31 @@ int getIntField(const BSONObj& b, const std::string& field)
 
 /* ****************************************************************************
 *
-* getBoolField -
+* getLongField -
+*/
+long getLongField(const BSONObj& b, const std::string& field)
```

Comment:
Great! So probably now we can replace all the usage of `getField(XXX).Long()` and `getField(XXX).numberLong()` usages in the code for this new function.

Probably not in this PR, but in a separted one. What do you think?